code/game/objects/items/defib.dm

```diff
@@ -292,6 +292,30 @@
 	var/grab_ghost = FALSE
 	var/tlimit = DEFIB_TIME_LIMIT * 10
 
+	var/datum/component/mobhook
+
+/obj/item/twohanded/shockpaddles/equipped(mob/user, slot)
+	. = ..()
+	if(req_defib)
+		if (mobhook && mobhook.parent != user)
+			QDEL_NULL(mobhook)
+		if (!mobhook)
+			mobhook = user.AddComponent(/datum/component/redirect, list(COMSIG_MOVABLE_MOVED), CALLBACK(src, .proc/check_range))
+
+/obj/item/twohanded/shockpaddles/Moved()
+	. = ..()
+	check_range()
+
+/obj/item/twohanded/shockpaddles/proc/check_range()
+	if(!in_range(src,defib))
+		var/mob/living/L = loc
+		if(istype(L))
+			to_chat(L, "<span class='warning'>[defib]'s paddles overextend and come out of your hands!</span>")
+			L.dropItemToGround(src)
```

Comment:
TRUE for force. Also you can use `temporarilyRemoveItemFromInventory` so it doesn't actually touch the ground and snaps right back (it will still call dropped)